src/sentry/backup/comparators.py

```diff
@@ -267,6 +279,17 @@ def truncate(self, data: list[str]) -> list[str]:
         return truncated
 
 
+class IgnoredComparator(JSONScrubbingComparator):
```

Comment:
Wondering what other use cases this would come handy for, are there many models with a hash as a field value?

Reply:
There are, but for these `HashObfuscatingComparator` is fine. `useremail` is something of a (hopefully!) one-off special case in that we want to merge it with existing data. Everything else is `INSERT`-only, but for `useremail`, we want to use the existing one for a particular email if we already have it.